src/CSharp/CodeCracker/Design/InconsistentAccessibility/InconsistentAccessibilityInMethodParameter.cs

```diff
@@ -8,11 +8,14 @@
 using System.Threading.Tasks;
 using System.Text.RegularExpressions;
 using System.Globalization;
+using CodeCracker.Properties;
 
 namespace CodeCracker.CSharp.Design.InconsistentAccessibility
 {
     public sealed class InconsistentAccessibilityInMethodParameter : InconsistentAccessibilityInfoProvider
     {
+        private static readonly LocalizableString CodeActionMessage = new LocalizableResourceString(nameof(Resources.InconsistentAccessibilityInMethodParameter_CodeActionMessage), Resources.ResourceManager, typeof(Resources));
```

Comment:
The correct name for the resource would be `InconsistentAccessibilityInMethodParameter_Title`.